js/register.js

```diff
@@ -0,0 +1,81 @@
+function PasswordChecker() { //ich benötige ID von:  PasswortFeld, PasswortFeldUeberprüfung, SubmitButton
```

Comment:
Register Sachen entfernen wenn nicht in diesem Branch fertiggestellt - halbfertiges soll **nie** in den master Branch kommen.